### PR TITLE
Option to disable logging of dechunking in audit log

### PR DESCRIPTION
**Pull request by marcstern**
when log level < 9.
[Issue #1068 - Marc Stern]

### CODE REVIEW

**Comment by zimmerle** [comment]
Please make sure that the JSON audit log is also covered.

**Comment by marcstern** [comment]
done

File: apache2/msc_logging.c [hunk 1]
```diff
@@ -1,2282 +1,2285 @@
-/*
```
Comment (zimmerle) [inline apache2/msc_logging.c]: This is a huge diff :) Please make sure to not make any change in the tabulation or in the line terminator, otherwise it is very difficult to read the patch.

File: CHANGES [hunk 2]
```diff
@@ -1,7 +1,7 @@
 DD MMM YYYY - 2.9.2 - To be released
 ------------------------------------
 
- * {dis|en}able-dechunking: Option to disable logging of
```
Comment (zimmerle) [inline CHANGES]: This is a very simple an compact patch. Please avoid to make a commit to fix your own commit. Use ```git commit --amend``` instead.

**Comment by zimmerle** [comment]
I am closing this pull request. There are comments on the patches.
